charts/istio/istio-istiod/templates/configmap.yaml

```diff
@@ -59,7 +59,7 @@ data:
     # If true, automatically configure client side mTLS settings to match the corresponding service's
     # server side mTLS authentication policy, when destination rule for that service does not specify
     # TLS settings.
-    enableAutoMtls: false
+    enableAutoMtls: true
```

Comment:
Isn't this true by default?

Reply:
It's from `1.6` https://github.com/istio/istio/blob/418e60eda506dfc61a9e333480096b6a6f7756ed/pkg/config/mesh/mesh.go#L156

It's better to be explicit and set it to `true`